test/cases/getSavedTimeForUser.js

```diff
@@ -1,6 +1,6 @@
 var request = require('request');
 var utils = require('../utils.js');
-var db = require('../../src/databases/databases.js').db;
+var db = require('../../src/databases/databases.js').db.getConnection();
```

Comment:
Why does this test use the connection?

Reply:
i resolved test errors differently in one file before i'd made the service and just exposed the db object - i'll change it to keep it inline with the other test files.